Type definitions: ExtendedAttachment images Array property invalid (possibly TS 4.x related)

I suspect this is an issue after we recently moved to using TypeScript `4.x` for our project compilation as it's not something I noticed earlier, and notice that the change was introduced 19 days ago as part of 0448d3474f712d78d557980f94c66e78e2d0c2f4.

```
$ tsc --noEmit
node_modules/stream-chat-react/types/index.d.ts:692:12 - error TS2314: Generic type 'Array<T>' requires 1 type argument(s).

692   images?: Array;
```

gz#6310

--- a/src/components/Attachment/Attachment.js
+++ b/src/components/Attachment/Attachment.js
@@ -127,7 +127,7 @@ export const renderAttachmentActions = (props) => {
 export const renderGallery = (props) => {
   const { attachment: a, Gallery } = props;
   return renderAttachmentWithinContainer(
-    <Gallery images={a.images} key="gallery" />,
+    <Gallery images={a.images || []} key="gallery" />,
     a,
     'gallery',
   );

--- a/src/components/ChannelList/hooks/usePaginatedChannels.js
+++ b/src/components/ChannelList/hooks/usePaginatedChannels.js
@@ -4,7 +4,7 @@ import { useEffect, useState } from 'react';
 import { MAX_QUERY_CHANNELS_LIMIT } from '../utils';
 /**
  * @typedef {import('stream-chat').Channel} Channel
- * @param {import('stream-chat').StreamChat} client
+ * @param {import('types').StreamChatReactClient} client
  * @param {import('stream-chat').ChannelFilters} filters
  * @param {import('stream-chat').ChannelSort} [sort]
  * @param {import('stream-chat').ChannelOptions} [options]

--- a/src/components/ChannelList/utils.js
+++ b/src/components/ChannelList/utils.js
@@ -22,7 +22,7 @@ export const moveChannelUp = (cid, channels) => {
 };
 
 /**
- * @param {import('stream-chat').StreamChat} client
+ * @param {import('types').StreamChatReactClient} client
  * @param {string} type
  * @param {string} id
  */

--- a/types/index.d.ts
+++ b/types/index.d.ts
@@ -691,7 +691,10 @@ export interface ExtendedAttachment extends Client.Attachment {
   id?: string;
   asset_url?: string;
   mime_type?: string;
-  images?: Array;
+  images?: Array<{
+    image_url?: string;
+    thumb_url?: string;
+  }>;
 }
 
 export interface BaseAttachmentUIComponentProps {

